screenpipe-server/src/bin/screenpipe-server.rs

```diff
@@ -48,10 +48,21 @@ const DISPLAY: &str = r"
 ";
 
 fn get_base_dir(custom_path: &Option<String>) -> anyhow::Result<PathBuf> {
-    let default_path = home_dir()
+    let mut default_path = home_dir()
         .ok_or_else(|| anyhow::anyhow!("failed to get home directory"))?
         .join(".screenpipe");
 
+    // check .screenpipe_dir file whether it exists in default_path, if so, replace default_path
+    let screenpipe_dir_path = default_path.join(".screenpipe_dir");
```

Comment:
what is `.screenpipe_dir`? don't recall ever using this dir

<img width="1190" alt="Screenshot 2024-11-14 at 9 26 54 AM" src="https://github.com/user-attachments/assets/230c83c6-3bd5-4bba-afba-1dff091a6df4">

this is mine with default data dir